bridge/engine/ActiveMediaList.cpp

```diff
@@ -130,14 +130,24 @@ bool ActiveMediaList::onAudioParticipantAdded(const size_t endpointIdHash, const
         _dominantSpeakerId = endpointIdHash;
     }
 
+    if (_audioSsrcRewriteMap.size() == _audioLastN)
```

Comment:
This is important. We will have 2 more ssrcs available than the _audioLastN. We can't allow  to go beyond _audioSsrcRewriteMap.

If you don't limit this, it can grows until 5 and we can have until 10 audio changes on bb (5 endpoints removed and other new 5 added) but the map only has space to 8.